app/Dockerfile

```diff
@@ -1,18 +1,13 @@
-# Use the Active LTS version of Node.
-# See https://nodejs.org/en/about/releases/
-FROM node:16-alpine
-# Keep container packages up-to-date.
-# -U runs both apk update and apk upgrade.
-RUN apk -U upgrade
-# Copy just the package data to the working directory.
-COPY yarn.lock /srv
-COPY package.json /srv
-WORKDIR /srv
-# Install application dependencies.
-RUN yarn install --frozen-lockfile
-# Copy all the remaining application files (ignoring files in .dockerignore) to the working directory.
-COPY . /srv
-# Build the application.
-RUN yarn build
-# Run the application.
-CMD yarn start
+# Simple hello, world image mostly copied
+# from https://github.com/crccheck/docker-hello-world
+FROM busybox:latest AS release
+ENV PORT=8000
+
+ADD index.html /www/index.html
+
+# EXPOSE $PORT
```

Comment:
Should we remove or uncomment this?

Reply:
i'll uncomment. not sure why it was commented in the first place in the reference file